### PR TITLE
Manage forms - files to tailwind config colors

### PR DESCRIPTION
**Pull request by sandramsc**
## Change color assignments to **tailwind.config.js** colors:

Discussed in live stream and GitHub Discussion https://github.com/EddieHubCommunity/LinkFree/discussions/7770

- `text-gray-400` (any color or value after the color is incorrect way and should be using text-primary-medium )

- same with` bg-white` (using the color is incorrect)

- the available colors in our project are here in the **tailwind config** under colors https://github.com/EddieHubCommunity/LinkFree/blob/main/tailwind.config.js 

-  we should not use any of these text-[color] or text-[color]-[value] or with bg also like bg-[color] or bg-[color]-[value]

-------------------------------------------------------------------------------------
**Continued with color updates in files**.

**Pages fixed from 1st related PR**:
username.js

Before: icon

After: icon

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7791"></a>

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Thanks @sandramsc 👍 Another PR got merged that had light/dark theme changes also. Could you resolve the conflicts, thanks

File: pages/map.js
Comment (yoyoekta) [inline pages/map.js:1]: I don't understand the change. Isn't it the same?

Comment (sandramsc) [reply pages/map.js:1]: Lower case changed to upper case to maintain consistency.

Comment (yoyoekta) [reply pages/map.js:1]: Good spot! Thanks for clarifying.

File: components/GitHubAccelerator.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ export default function GitHubAccelerator() {
             <div className="mt-6 flex flex-col justify-stretch space-y-3 sm:flex-row sm:space-x-4 sm:space-y-0">
               <Link
                 href="http://accelerator.github.com"
-                className="inline-flex justify-center rounded-md bg-white px-3 py-2 text-sm font-semibold text-primary-high dark:text-primary-high shadow-sm ring-1 ring-inset ring-primary-low-medium hover:bg-primary-low dark:hover:bg-slate-200 cursor-pointer"
+                className="inline-flex justify-center rounded-md bg-primary-low px-3 py-2 text-sm font-semibold text-primary-high dark:text-primary-high shadow-sm ring-1 ring-inset ring-primary-low/80 hover:bg-primary-low/95 dark:hover:bg-primary-low/95 cursor-pointer"
```
Comment (yoyoekta) [inline components/GitHubAccelerator.js:38]: Could you please tell what does this mean and how you come up with this? I want to know more about `/80` and `/95` thing.

`ring-primary-low-medium` to `ring-primary-low/80`
`bg-slate-200` to `bg-primary-low/95`

**Review by yoyoekta — APPROVED** [review]

File: components/GitHubAccelerator.js (hunk 1 above)
Comment (sandramsc) [reply components/GitHubAccelerator.js:38]: Sure, I studied the previous code and understood the the logic behind the styling first, then implemented the change as per the tailwind.config.js requirements.

i.e 
**before**: `bg-white/5` , changing the numerical value in turn changes the shade of the color preceding it.

**after**: `bg-primary-low/85` (values starting ~ 5 to 100 will give different shades of the color associated with -primary-low (or any of the configured colors) in the tailwind.config.js file)

Hope that helps.

File: pages/newsletter.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ export default function Newsletter() {
                     </div>
                   </div>
                   <div className="flex flex-col items-start">
-                    <div className="rounded-md bg-primary-low dark:bg-primary-medium p-2 ring-1 ring-primary-medium">
+                    <div className="rounded-md bg-primary-low/25 dark:bg-primary-medium dark:ring-primary-medium-low p-2 ring-1 ring-primary-low/100">
```
Comment (ChinmayMhatre) [inline pages/newsletter.js:63]: I feel the border around both icons can be a better darker in light mode

File: components/Modal.js
Comment (ChinmayMhatre) [inline components/Modal.js:1]: image
The close icon is not visible in light mode and the section below should be a different gray

File: components/IconCard.js
Comment (ChinmayMhatre) [inline components/IconCard.js:1]: image
Icon names not visible in light mode

File: components/IconCard.js [hunk 3]
```diff
@@ -8,15 +8,15 @@ export default function IconCard({ iconName }) {
   return (
     <>
       <button
-        className="border-2 p-2 border-slate-100 w-24 h-24 flex flex-col items-center justify-around rounded hover:border-secondary-high hover:cursor-pointer active:border-green-600 focus:border-green-600"
+        className="border-2 p-2 border-primary-low/95 w-24 h-24 flex flex-col items-center justify-around rounded hover:border-secondary-high hover:cursor-pointer active:border-green-600 focus:border-green-600"
```
Comment (ChinmayMhatre) [inline components/IconCard.js:11]: border can just be primary-low?

File: components/GitHubAccelerator.js (hunk 1 above)
Comment (ChinmayMhatre) [reply components/GitHubAccelerator.js:38]: It changes the alpha of the colour which is the opacity not the actual shade. so `green-50/90` is `green-50` with 90 percent opacity

Comment (ChinmayMhatre) [reply components/GitHubAccelerator.js:38]: you can learn more here https://tailwindcss.com/docs/text-color . under the changing opacity section

File: components/GitHubAccelerator.js
Comment (ChinmayMhatre) [inline components/GitHubAccelerator.js:1]: image
The border around the buttons are not visible

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Hi @sandramsc , Thank you for your effort and awesome work on the PR!🔥 I have left a few comments where the pages are breaking and a few other comments. Do check it out and let me know.

File: components/GitHubAccelerator.js (hunk 1 above)
Comment (sandramsc) [reply components/GitHubAccelerator.js:38]: I didn't realize there was a difference between shade and opacity.

File: pages/newsletter.js (hunk 2 above)
Comment (sandramsc) [reply pages/newsletter.js:63]: This opacity/shade was chosen to replicate the previous color `bg-white/5`. The goal of this PR is to replace the bg-[color] with its closest tailwind.config.js configuration. The comment above leans more towards changing the original design. I hope I understood your request correctly.
svg

Comment (ChinmayMhatre) [reply pages/newsletter.js:63]: You have changed the ring colour from `primary-medium` to `primary-low/100` which isn't required . you can see this on line 63 as marked by the comment. same is the case which `bg-primary-low`

Comment (sandramsc) [reply pages/newsletter.js]: - This change was made to fix the color adjustments from the first pull request within the same issue. Consider viewing the discussion here: https://github.com/EddieHubCommunity/LinkFree/pull/7782#discussion_r1241006010
- I've updated the opacity/shade again to colors that more closely replicate the original colors.
- My choices for the color choice is again purely to replicate the original (before) colors as close as possible to the colors available in the tailwind.config.js
- Here is how the colors look in my code editor, when comparing the **before** and **after** (hence the choice)
newletter_icon

Comment (sandramsc) [reply pages/newsletter.js]: - This change was made to fix the color adjustments from the first pull request within the same issue. Consider viewing the discussion here: https://github.com/EddieHubCommunity/LinkFree/pull/7782#discussion_r1241006010
- I've updated the opacity/shade again to colors that more closely replicate the original colors.
- My choices for the color choice is again purely to replicate the original (before) colors as close as possible to the colors available in the tailwind.config.js
- Here is how the colors look in my code editor, when comparing the **before** and **after** (hence the choice)
newsletter_icons

**Pull request closed without merging**